pkg/util/version/const.go

```diff
@@ -76,19 +76,19 @@ var (
 
 // FluentbitImage contains the location of the Fluentbit container image
 func FluentbitImage(acrDomain string) string {
-	return acrDomain + "/fluentbit:1.9.9-1"
+	return acrDomain + "/fluentbit:1.9.10-cm20230321"
 }
 
 // MdmImage contains the location of the MDM container image
 // https://eng.ms/docs/products/geneva/collect/references/linuxcontainers
 func MdmImage(acrDomain string) string {
-	return acrDomain + "/distroless/genevamdm:mariner_20221026.2"
+	return acrDomain + "/genevamdm:2.2023.331.1521-399d45-20230331t1638"
 }
 
 // MdsdImage contains the location of the MDSD container image
 // see https://eng.ms/docs/products/geneva/collect/references/linuxcontainers
 func MdsdImage(acrDomain string) string {
-	return acrDomain + "/genevamdsd:master_20221018.2"
+	return acrDomain + "/genevamdsd:mariner_20230413.1"
```

Comment:
Shouldn't this be `mariner_20230418.1` (according to the sourced doc)?

Reply:
That updated since I did the mirror :D